examples/demo_mult_image.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+# title           : this_python_file.py
```

Comment:
In this example (and others like it) can you drop the word `demo` from the title